index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="https://kit.fontawesome.com/0bd1156d30.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="Main-Page">
+     <nav class="nav">
+<div class="logo">AFIAMOAH</div>
+
+<div class="nav-links-menu"><p class="side-logo"><i class="fa-solid fa-bars"></i></p></div>
+
+     </nav>
+     <section class="section">
+<h1>Hey There. I’m Afiamoah</h1>
+<h2>I’am a Software Developer</h2>
+<p> class="content">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.</p>
+    <div class="imgcontainer">
+        <div class="images"><img src="linkedin.png" alt=""></div>
+        <div class="images"><img src="Enabled.png" alt=""></div>
+        <div class="images"><img src="twitter.png" alt=""></div>
+        <div class="images"><img  id="Medium" src="Medium.png" alt=""></div>
+        
+    </div>
+    <div class="display">
+
+    </div>
+    </section>
+   
+    
+    <!-- <div><p class="side-logo"><i class="fa fa-github" aria-hidden="true"></i><i class="fab fa-linkedin"></i><i class="fab fa-angellist"></i><i class="fa fa-twitter" aria-hidden="true"></i><img src="Icon - Medium"> -->
```

Comment:
- [ ] Kindly refrain from keeping commented-out code in your files since it is considered bad practice and makes the code less maintainable and more cluttered.